doc/tutorials/cascading.md

```diff
@@ -0,0 +1,131 @@
+## Order of namespaces
+
+This tutorial assumes that you know what namespaces are. You might want to read
+the [Namespaces tutorial](/doc/tutorials/namespaces.md) first.
+
+In Elektra, the default order of namespaces is as follows:
+
+ * [spec](https://github.com/ElektraInitiative/libelektra/blob/master/doc/help/elektra-namespaces.md#spec) (modify elektra lookup behaviour)
+ * [proc](https://github.com/ElektraInitiative/libelektra/blob/master/doc/help/elektra-namespaces.md#proc) (process-related information)
+ * [dir](https://github.com/ElektraInitiative/libelektra/blob/master/doc/help/elektra-namespaces.md#dir) (directory-related information, e.g. `.git` or `.htaccess`)
+ * [user](https://github.com/ElektraInitiative/libelektra/blob/master/doc/help/elektra-namespaces.md#user) (user configuration)
+ * [system](https://github.com/ElektraInitiative/libelektra/blob/master/doc/help/elektra-namespaces.md#system) (system configuration)
+ * [cascading](https://github.com/ElektraInitiative/libelektra/blob/master/doc/help/elektra-namespaces.md#cascading) (keys without a namespace)
+
+Looking at this order, we can see that if a configuration option isn't specified
+by the user (then it would be in the `user` namespace), it will be loaded from
+the `system` namespace. In this case, the option in the `system` namespace works
+like a default value.
+
+```
+$ sudo kdb set system/t/test "hello world"
+create a new key system/t/test with string hello world
+
+$ kdb get /t/test
+hello world
+
+$ kdb set user/t/test "hello universe"
+Create a new key user/t/test with string hello universe
+
+$ kdb get /t/test
+hello universe
+```
+
+Furthermore, we can see that `dir` is even higher than `user`, which means that
+configuration in the current folder can overwrite user configuration.
+
+`.configuration` in your current directory:
+
+```
+test = hello dir
+```
+
+Then run:
+
+```
+$ sudo kdb mount /.configuration dir/t ini
+$ kdb get /t/test
+hello dir
+```
+
+
+## Cascading
+
+Cascading triggers secondary actions when, for example, the key isn't found.
```

Comment:
Cascading is one way how a lookup can be done, its not really a secondary action, but a different action.